src/send_and_confirm.rs

```diff
@@ -115,8 +115,7 @@ impl Miner {
                 // Resign the tx
                 let (hash, _slot) = client
                     .get_latest_blockhash_with_commitment(self.rpc_client.commitment())
-                    .await
-                    .unwrap();
+                    .await?;
```

Comment:
This file is the only one we should commit. Can you revert your other changes? The comments you removed above are helpful.